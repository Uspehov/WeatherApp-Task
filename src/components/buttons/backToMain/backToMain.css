.backToMain-button {
    height: 10%;
    background-color: cornflowerblue;
    text-align: center;
    font-size: 23px;
    margin: auto;
    margin-top: 40px;
    margin-bottom: 40px;
}