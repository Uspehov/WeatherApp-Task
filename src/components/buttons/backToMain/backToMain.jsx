import React from 'react';
import { Link } from "react-router-dom";
import './backToMain.css';

export function BackToMainButton() {
    return (
        <Link replace={true} to={`/`} className={`backToMain-button mainButton`}>Go Back</Link>
    )
}
