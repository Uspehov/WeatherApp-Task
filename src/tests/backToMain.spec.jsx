import React from 'react';
import  { shallow, configure }  from 'enzyme';
import { BackToMainButton } from "../components/buttons/backToMain/backToMain.jsx";
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter()});

describe('Baack to Main button', () => {

    it('should render', () => {
        const wrapper = shallow(<BackToMainButton />);
        expect(wrapper.debug()).toMatchSnapshot();
    });

});

