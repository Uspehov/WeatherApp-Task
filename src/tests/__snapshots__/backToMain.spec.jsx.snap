// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Baack to Main button should render 1`] = `
"<Link replace={true} to=\\"/\\" className=\\"backToMain-button mainButton\\">
  Go Back
</Link>"
`;
